package.json

```diff
@@ -23,7 +23,7 @@
 		"build": "lerna run build",
 		"watch": "lerna run watch --no-sort --stream",
 		"start": "pm2 start ecosystem.config.js --no-daemon",
-		"docs": "docsify serve -p 3100 ./",
+		"docs": "docsify serve -p 3100 ./docs",
```

Comment:
@ken why are we still building docs if you removed them? Can this line go away?

Reply:
Yep. Removed